files/kubelet/scripts/launch-edgenet.sh

```diff
@@ -16,5 +16,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ----------------------------------------------------------------------------
+ENV_EDGENET_SUBNET=$(snapctl get kubelet.edgenet-subnet)
+ENV_EDGENET_GATEWAY=$(snapctl get kubelet.edgenet-gateway)
 
-docker network inspect edgenet &>/dev/null || docker network create --subnet=10.0.0.0/24 --gateway=10.0.0.1 edgenet
+if [[ ${ENV_EDGENET_SUBNET} = "" ]]; then
+	EDGENET_SUBNET=10.0.0.0/24
+else
+	EDGENET_SUBNET=${ENV_EDGENET_SUBNET}
+fi
+
+if [[ ${ENV_EDGENET_GATEWAY} = "" ]]; then
+	EDGENET_GATEWAY=10.0.0.1
+else
+	EDGENET_GATEWAY=${ENV_EDGENET_GATEWAY}
```

Comment:
There is a special form meant exactly for this case. Instead of the entire `if... else...fi`, please use:
`EDGENET_GATEWAY=${ENV_EDGENET_GATEWAY:-10.0.0.1}`